Lil-Peep/src/sections/featured-kits.liquid

```diff
@@ -0,0 +1,202 @@
+<section class="featured-kits">
+  <div class="flex-grid">
+    {% for block in section.blocks %}
+      {% if block.settings.opc == "1" %}
+        <div class="big-banner banner-{{ block.id }}">
+          <div class="big-banner-content">
+            <h1 class="big-banner-title">{{ block.settings.title }}</h1>
+            <p class="big-banner-description">{{ block.settings.description }}</p>
+            <div class="view-in-detail">
+              <a href="#" >
+                <h4 class="view-detail-button">{{ block.settings.button }}</h4>
+              </a>
+            </div>
+          </div>
+        </div>
+      {% else %}
+        {% if block.settings.opc == "2"%}">
+          <div class="small-banner banner-{{ block.id }}">
+            <p class="small-banner-tittle">{{ block.settings.title }}</p>
+              <div class="big-text">
+                <p class="small-banner-description">{{ block.settings.description }}</p>
+                <p class="small-banner-description">{{ block.settings.button }}</p>
+             </div>
+         </div>
+       {% endif %}
+      {% endif %}
+    {% endfor %}
+  </div>
+  <div class="flex-grid">
+    {% for block in section.blocks %}
+      {% if block.settings.opc == "3"  %}
+              <div class="mid-banner mid-banner-saylor banner-{{ block.id }}">
+                <div class="mid-banner-content">
+                  <h1>{{ block.settings.title }}</h1>
+                  <p class="description">{{ block.settings.description }}</p>
+                  <button class="button-mid-banner button-sailor" type="button" name="button">
+                    <h1 class="h1-sailor">{{ block.settings.button }}</h1>
+                  </button>
+                </div>
+              </div>
+       {% else %}
+        {% if block.settings.opc == "4" %}
+          <div class="mid-banner mid-banner-archeologist banner-{{ block.id }}">
+            <div class="mid-banner-content">
+              <h1>{{ block.settings.title }}</h1>
+              <p class="description">{{ block.settings.description }}</p>
+              <button class="button-mid-banner button-archeologist" type="button" name="button">
+                <h1 class="h1-archeologist">{{ block.settings.button }}</h1>
+              </button>
+            </div>
+          </div>
+        {% endif %}
+      {% endif %}
+    {% endfor %}
+  </div>
+</section>
+
+<style>
+{% for block in section.blocks %}
+  .banner-{{block.id}} {
```

Comment:
Remember to add a space after and before the curly braces like `{{ block.id }}` so that it could be easy to read.